tests/test_cookie_storage_with_bson.py

```diff
@@ -0,0 +1,179 @@
+import asyncio
+import bson.json_util as json
+from bson import objectid
+import socket
+import unittest
+import time
+
+from aiohttp import web, request
+from aiohttp_session import (Session, session_middleware,
+                             get_session, SimpleCookieStorage)
+
+
+class TestSimleCookieStorage(unittest.TestCase):

```

Comment:
Thanks for the feedback. this is my first time writing testcase.
I also thought like you but this is not completely copy / paste.
When I declared `from import bson json_util as json`, the creation of additional test file will make the test simpler and more explicit. Besides, I do not really understand the logic of the session changed. So I chose to copy / paste.
I'll fix it in the next commit.